### PR TITLE
placeholder for ISC23

### PR DESCRIPTION
**Pull request by gflofst**

### CODE REVIEW

File: templates/Pages/bof_isc23.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?php $this->assign('title', 'Birds-of-a-Feather - ISC 23'); ?>
+
+<nav id="breadcrumb">
+    <p>YOU ARE HERE</p>
+
+    <?php
+    $this->Breadcrumbs->add(__('HOME'), ['controller' => 'submissions', 'action' => 'latest']);
+    $this->Breadcrumbs->add(__('BOF'), ['controller' => 'pages', 'action' => 'display', 'bofs']);
+    $this->Breadcrumbs->add(__("SC'22"), ['controller' => 'pages', 'action' => 'display', 'bof-sc22']);
```
Comment (jeanbez) [inline templates/Pages/bof_isc23.php:9]: We also need to update here to point to this page

File: templates/Pages/bof_isc23.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    echo $this->Breadcrumbs->render([], ['separator' => ' / ']);
+    ?>
+</nav>
+
+<div class="content">
+    <h2>SC'22</h2>
```
Comment (jeanbez) [inline templates/Pages/bof_isc23.php:16]: We also need to update here

**Review by jeanbez — CHANGES_REQUESTED** [review]

File: templates/Pages/bof_isc23.php (hunk 1 above)
Comment (gflofst) [reply templates/Pages/bof_isc23.php]: done

File: templates/Pages/bof_isc23.php (hunk 2 above)
Comment (gflofst) [reply templates/Pages/bof_isc23.php]: done

**Review by gmarkomanolis — APPROVED** [review]

**Pull request merged**